src/uTensor/core/types.hpp

```diff
@@ -1,7 +1,11 @@
 #ifndef __UTENSOR_TYPES_H
 #define __UTENSOR_TYPES_H
+
+// https://www.arduinolibraries.info/libraries/avr_stl
 #include <array>
-#include <cstdint>
+// check if the follow modificiation affect other builds
+//#include <cstdint>
+#include <stdint.h>
```

Comment:
Does arduino not have cstdint? usually stdint and cstdint are different slightly

Reply:
That's right. Hopefully, this doesn't kill the precision.